<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro_estoque</class>
 <widget class="QDialog" name="cadastro_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Cadastro Estoque</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_cad_estoque">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>741</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Nome Estoque</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_cad_nome_estoque"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Setor Estoque</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_cad_setor_estoque"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Nome Localização</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_cad_nome_localizacao"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Quantidade máxima na localização</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_cad_quant_max_loc"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cad_estoque">
        <property name="text">
         <string>Cadastrar Estoque</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Consulta Estoque</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtrar Estoque</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="rb_filtrar_username">
            <property name="text">
             <string>Nome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_filtrar_cep">
            <property name="text">
             <string>Loc</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_filtrar_nome">
            <property name="text">
             <string>Setor</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_filtrar_usuario"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_filtrar_estoque">
          <property name="text">
           <string>Filtrar Estoque</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tableWidget_cad_estoque_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>500</width>
       <height>220</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>230</width>
       <height>220</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Nome Estoque</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_cad_nome_estoque_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Nome Localização</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_cad_nome_localizacao_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Setor Estoque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_setor_estoque_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Quantidade máxima na localização</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_cad_quant_max_loc_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_cad_estoque_2">
          <property name="text">
           <string>Alterar Estoque</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cad_estoque_3">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
